### PR TITLE
PR: Initial implementation to pass name instead of full directory for environments and list environments command

### LINKED ISSUE #10: Add `list_environments` command
* Add a way to config the root directory where all the backends will need to be created (change passing the whole path to the env to just name)
* Add command to list environments for a specific backend

### PR DESCRIPTION
**Pull request by dalthviz**
Fixes #10

### CODE REVIEW

**Comment by dalthviz** [comment]
Note: Make list_environments static and without needing to pass `env_name` or `env_directory` (just `backend` and `root_path`)

File: env_manager/manager.py [hunk 1]
```diff
@@ -16,11 +16,25 @@ class Manager:
...
-    def __init__(self, backend, env_directory, external_executable=None):
+    def __init__(
+        self,
+        backend,
+        root_path=None,
+        env_name=None,
+        env_directory=None,
+        external_executable=None,
+    ):
         backend_class = self.BACKENDS[backend]
```
Comment (ccordoba12) [inline env_manager/manager.py:27]:
```suggestion
        backend_class = self.BACKENDS[backend]

```

File: env_manager/manager.py [hunk 2]
```diff
@@ -16,11 +16,25 @@ class Manager:
...
-        self.env_directory = str(env_directory)
+        if env_directory:
+            self.env_directory = str(env_directory)
+        elif root_path and env_name:
+            self.env_directory = root_path / backend / "envs" / env_name
+        else:
+            raise Exception(
+                "'env_directory' or 'root_path' and 'env_name' should be provided"
+            )
         self.backend_instance = backend_class(
```
Comment (ccordoba12) [inline env_manager/manager.py:36]:
```suggestion

        self.backend_instance = backend_class(
```

**Review by ccordoba12 — COMMENTED** [review]
Looks good to me @dalthviz, except for the two minor style issues I left for you.

**Comment by ccordoba12** [comment]
> Note: Make list_environments static and without needing to pass env_name or env_directory (just backend and root_path)

Do you plan to address that here? Or do you want to leave it for a follow-up PR?

**Comment by dalthviz** [comment]
I think this is ready @ccordoba12

File: env_manager/manager.py [hunk 3]
```diff
@@ -84,12 +95,21 @@ def update(self, packages, force=False, capture_output=False):
...
+        print(f"# {backend} environments")
         for env_dir in envs_directory.iterdir():
             if env_dir.is_dir():
+                if first_environment:
+                    first_environment = False
                 environments[env_dir.name] = str(env_dir)
                 print(f"{env_dir.name} - {str(env_dir)}")
+        else:
+            if first_environment:
+                print(f"No environments found for {backend} on {root_path}")
```
Comment (ccordoba12) [inline env_manager/manager.py:114]:
```suggestion
                print(f"No environments found for {backend} in {root_path}")
```

**Review by ccordoba12 — COMMENTED** [review]
Another tiny suggestion for you @dalthviz, then it should be ready.

**Review by ccordoba12 — APPROVED** [review]
Thanks @dalthviz!

**Pull request merged**